components/schedule-editor/src/ScheduleEditor.svelte

```diff
@@ -116,21 +209,266 @@
       <input
         type="radio"
         name="show_hosts"
-        bind:group={manifestProperties.show_hosts}
-        value="show"
+        value={"show"}
+        on:input={(e) => {
+          manifestProperties.show_hosts = e?.target?.value;
+        }}
       />
       <span>Show Hosts</span>
     </label>
     <label>
       <input
         type="radio"
         name="show_hosts"
-        bind:group={manifestProperties.show_hosts}
-        value="hide"
+        value={"hide"}
+        on:input={(e) => {
+          manifestProperties.show_hosts = e?.target?.value;
+        }}
       />
       <span>Hide Hosts</span>
     </label>
   </div>
-
+  <div>
+    <label>
+      <strong>Start Hour</strong>
+      <input
+        type="range"
+        min={0}
+        max={24}
+        step={1}
+        value={0}
+        on:input={(e) => {
+          manifestProperties.start_hour = e?.target?.value;
```

Comment:
For some weird reason bind:value doesn't seem to be doing the job here. Having the `on:input` ensures the value in `manifestProperties.start_hour` is updated on input.

Reply:
`bind:value` style bindings should now be back via https://github.com/nylas/components/pull/80/commits/a7a0f6c1e95a764f5a475e9b3f84fffdb48cbea1

Reply:
Thank you, Phil!